mod/dfrn_notify.php

```diff
@@ -230,16 +188,96 @@ function dfrn_notify_post(App $a) {
 		logger('rino: decrypted data: ' . $data, LOGGER_DATA);
 	}
 
+	logger('Importing post from ' . $importer['addr'] . ' to ' . $importer['nickname'] . ' with the RINO ' . $rino_remote . ' encryption.', LOGGER_DEBUG);
+
 	$ret = DFRN::import($data, $importer);
 	System::xmlExit($ret, 'Processed');
 
 	// NOTREACHED
 }
 
+function dfrn_dispatch_public($postdata)
```

Comment:
Shouldn't this function be in `src/Protocol/DFRN.php` ?

Reply:
I was unsure about this, but had the same thought. But especially the preparation part of the private dispatcher has got some checks where it fails with some ```xmlExit``` which is surely frontend stuff.

There is no "business logic" inside these functions, only some preparation work, so I guess this will be fine.

Reply:
Ok.